src/Mod/Part/App/TopoShapeExpansion.cpp

```diff
@@ -551,6 +554,142 @@ TopoShape::makeElementCompound(const std::vector<TopoShape>& shapes, const char*
     return *this;
 }
 
+struct MapperFill: Part::TopoShape::Mapper
+{
+    BRepFill_Generator& maker;
+    MapperFill(BRepFill_Generator& maker)
+        : maker(maker)
+    {}
+    virtual const std::vector<TopoDS_Shape>& generated(const TopoDS_Shape& s) const override
+    {
+        _res.clear();
+        try {
+            TopTools_ListIteratorOfListOfShape it;
+            for (it.Initialize(maker.GeneratedShapes(s)); it.More(); it.Next()) {
+                _res.push_back(it.Value());
+            }
+        }
+        catch (const Standard_Failure& e) {
+            if (FC_LOG_INSTANCE.isEnabled(FC_LOGLEVEL_LOG)) {
+                FC_WARN("Exception on shape mapper: " << e.GetMessageString());
+            }
+        }
+        return _res;
+    }
+};
+
+// topo naming counterpart of TopoShape::makeShell()
+TopoShape& TopoShape::makeElementShell(bool silent, const char* op)
```

Comment:
I'd rather see this boolean as a `class enum` -- are there other methods in the new code that use this idea of "silent"?

Reply:
Yes, and there is a fair history of them across the code base.  Try PartDesign::ProfileBased as one example.  I'm not sure it makes sense to change this in just one module.  Could it be a separate, code base wide change someone makes?